mhw_analysis/ai/ai_mhw_images.py

```diff
@@ -0,0 +1,295 @@
+# imports
+from importlib import reload
+import os
+import numpy as np
+from matplotlib import pyplot as plt
+
+import iris
+
+import sys
+
+# create module paths
+MHW_PATH = '/Users/kamil/Desktop/research2020/mhw_analysis'
+OCEANPY_PATH = '/Users/kamil/Desktop/research2020/oceanpy'
+os.environ['NOAA_OI'] = '/Users/kamil/Desktop/NOAA-OI-SST-V2'
+
+# append paths
+sys.path.append(MHW_PATH)
```

Comment:
this shouldn't be necessary.  You can and should
run

`python setup.py develop`

for each of the Repo's.